app/Policies/AttendancePolicy.php

```diff
@@ -67,7 +78,7 @@ public function delete(User $user, Attendance $attendance)
      */
     public function restore(User $user, Attendance $attendance)
     {
-        return false;
+        return $user->hasRole('admin');
```

Comment:
role check to discuss

Reply:
Leaving this as admin since users can't create attendance in Nova yet